yb-voyager/src/srcdb/ora2pg_extract_schema.go

```diff
@@ -83,13 +54,36 @@ func ora2pgExtractSchema(source *Source, exportDir string) {
 		}
 
 		err = exportSchemaObjectCommand.Wait()
+		var errMsgs []string
```

Comment:
Let's simplify even more. Just like the `export data` code path, log the STDOUT and STDERR at this point.
In the `else` block on line 62, all you need to do is check if `errbuf.String()` and `outbuf.String()` contain `"error"` (that you can check using `strings.Contains()`). If there is an `"error"`, just write 1 to the channel. No need of iterating it line-by-line.